lxqt-config-session/autostartmodel.cpp

```diff
@@ -150,7 +150,7 @@ bool AutoStartItemModel::removeRow(int row, const QModelIndex& parent)
 
     if (!mItemMap[item].removeLocal())
     {
-        QModelIndex index = parent.child(row, 0);
+        QModelIndex index = this->index(row, 0, parent);
```

Comment:
There's no need to `this->`.

Reply:
Sure!
I put it there just because of the variable name :)

Reply:
Oh, yes! Please change the variable name to `indx`.